alma/alma-8.x/alma-8.6-hpc/README.md

```diff
@@ -0,0 +1,35 @@
+# AlmaLinux 8.6 HPC Image
+
+The AlmaLInux 8.6 HPC Image includes optimizations and recommended configurations to deliver optimal performance,
+consistency, and reliability. This image consists of the following HPC tools and libraries:
+
+- Mellanox OFED
+- Pre-configured IPoIB (IP-over-InfiniBand)
+- Popular InfiniBand based MPI Libraries
+  - HPC-X
+  - IntelMPI
+  - MVAPICH2
+  - OpenMPI
+- Communication Runtimes
+  - Libfabric
```

Comment:
Could you point me in which commit we install Libfabric?

Reply:
Libfabric is coming from IMPI installation

Reply:
IMPI tweaks Libfabric code and build separate providers for its own usage. It's a bit weird to count it. 
But I see we also list OpenUCX here. Do we list Libfabric in README of other distros?

Reply:
Yes, we list it for other distros as well
Ref - https://github.com/Azure/azhpc-images/tree/master/ubuntu/ubuntu-18.x/ubuntu-18.04-hpc#:~:text=Communication%20Runtimes-,Libfabric,-OpenUCX
https://github.com/Azure/azhpc-images/tree/master/centos/centos-7.x/centos-7.9-hpc#:~:text=Communication%20Runtimes-,Libfabric,-OpenUCX

Reply:
Ok. Let's leave as it is and keep it unified across different distros then.